python-package/pyproject.toml

```
@@ -1,6 +1,7 @@
 [build-system]
 requires = [
-    "hatchling>=1.12.1"
+    "hatchling>=1.12.1",
+    "pip>=22.0",
```

Appears `packaging` added `platform_tags` to the public API in commit ( https://github.com/pypa/packaging/commit/ba07d8287b4554754ac7178d177033ea3f75d489 ), which is part of [21.1]( https://github.com/pypa/packaging/releases/tag/21.1 )

```suggestion
    "packaging>=21.1",
```